lib/menu.rb

```diff
@@ -0,0 +1,28 @@
+class Menu
+
+  def initialize(display)
+    @display = display
+  end
+
+  def player_choice
+    @display.player_choice
+    potential_choice = @display.retrieve_user_input
+    choice = check_game_choice(potential_choice)
```

Comment:
Remember that Ruby returns the last line of a function. So you can just write `check_game_choice(...)`